src/engine/parsers/SquadronFactory.java

```diff
@@ -93,12 +93,13 @@ public SquadronFactory(String pathToFile, boolean renderSquadrons) throws FileNo
                         break;
                     case "Points": this.points = ParsingUtils.parseInteger("Points", value);
                         break;
-                    case "DefenseTokens": if(value.equals("None")){
-                            this.defenseTokens = new ArrayList<>();
+                    case "DefenseTokens":
+                        ArrayList<DefenseToken> tokens = new ArrayList<>();
+
+                        if(value.equals("None")){
+                            this.defenseTokens = tokens;
```

Comment:
I think lines 100 and 107 are identical duplicates and can just exist outside/after the if/else block. We should also be able to say "This if block does nothing" and just jump straight to the not "None" case

Reply:
Good catch.  Fixed